datamol/__init__.py

```diff
@@ -124,3 +124,7 @@
 from .viz import to_image
 
 from .mcs import find_mcs
+
+from .match import mol_to_nx_graph
```

Comment:
I would potentially rename the file name to prevent confusion around substruct match too.

Reply:
What about putting the functions in the `graph.py` file?

Reply:
Done